src/ProductConstructionService/ProductConstructionService.DependencyFlow/PullRequestUpdater.cs

```diff
@@ -1106,10 +1107,11 @@ private async Task CreateCodeFlowPullRequestAsync(
         string prBranch)
     {
         IRemote darcRemote = await _remoteFactory.CreateRemoteAsync(targetRepository);
+        var build = await _barClient.GetBuildAsync(update.BuildId);
         try
         {
-            var title = await _pullRequestBuilder.GenerateCodeFlowPRTitleAsync(update, targetBranch);
-            var description = await _pullRequestBuilder.GenerateCodeFlowPRDescriptionAsync(update, previousSourceSha);
+            var title = _pullRequestBuilder.GenerateCodeFlowPRTitle(targetBranch, new List<string> { update.SourceRepo });
+            var description = _pullRequestBuilder.GenerateCodeFlowPRDescription(update, build, previousSourceSha, null);
```

Comment:
```suggestion
            var description = _pullRequestBuilder.GenerateCodeFlowPRDescription(update, build, previousSourceSha, currentDescription: null);
```
think this improves the readability a lot